.github/workflows/pipeline.yml

```diff
@@ -74,15 +74,30 @@ jobs:
         run: |
           bundle exec rspec
 
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
 
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          # runs 'bundle install' and caches installed gems automatically
+          bundler-cache: true
+
+      - name: Rubocop
+        run: |
+          bundle exec rubocop
```

Comment:
Is it possible to have this do it with the `-A` flag to fix any automatically fixable linting issues?

Reply:
Don't think a pipeline is the best place for that since it would need to also commit code changes and then re-run. Would work as a local pre-commit hook though?